src/component/FooterComponent/SocialmediaIcons/SocialmediaIcons.jsx

```diff
@@ -0,0 +1,56 @@
+import IconButton from '@material-ui/core/IconButton';
+import FacebookIcon from '@material-ui/icons/Facebook';
+import InstagramIcon from '@material-ui/icons/Instagram';
+import PinterestIcon from '@material-ui/icons/Pinterest';
+import TwitterIcon from '@material-ui/icons/Twitter';
+import SocialLinks from '../Links/Social';
+import useStyles from './socialmediaStyle';
+
+const Socialmedia = () => {
+  const classes = useStyles();
+  const { instagram, facebook, twitter, pinterest } = SocialLinks;
+
+  return (
+    <div className={classes.socialIconStyle}>
+      <IconButton
```

Comment:
there's a lot of code repetition here (line 15-21 repeats a lot of times and it just changes the href).

I would rather have the SocialLinks be an array instead of an object, and then you can use the map function on each element of the array to create the IconButton.